internal/collectors/cron.go

```
@@ -603,8 +599,15 @@ func statusForOccurrence(when time.Time) string {
 	return "planned"
 }
 
-func (c *CronCollector) upsertJob(job CronJob) error {
-	_, err := c.db.Exec(
+func (c *CronCollector) upsertJobs(jobs []CronJob) []string {
+	if len(jobs) == 0 {
+		return nil
+	}
+	tx, err := c.db.Begin()
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Begin must not be called. use (*database/sql.DB).BeginTx

(noctx)